API/Controllers/MemoryUpdateController.js

```diff
@@ -0,0 +1,22 @@
+import {updateMemory} from '../Helpers/query';
+
+class MemoryUpdateController {
+
+    static updateMemory(request, response) {
+
+        const id = request.body.id
+        const title = request.body.title;
+        const mood = request.body.mood;
+        const story = request.body.story;
+        const picture = request.body.picture;
+      
+        updateMemory(id, title, story, mood, picture);
+
+        return response.status(201).json({
```

Comment:
You return a 200 when updating an existing record. You only use 201 when creating a new record
```suggestion
        return response.status(200).json({
```